ex1b.sh

```diff
@@ -0,0 +1,35 @@
+#1
+ls - S
+
+#1a
+ls -S /etc/ | head -1 

```

Comment:
The command is correct, but the trailing slash / after /etc is unnecessary. Remove the slash for the command to work properly